charts/incubator/zammad/values.yaml

```diff
@@ -0,0 +1,80 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: tccr.io/truecharts/zammad
+  tag: latest@sha256:c9c8ec07af85ca06ff2776e1bd00c70cca470c7bf9aa92c716b78283bf880f49
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsNonRoot: false
+    runAsGroup: 0
+    runAsUser: 0
+
+service:
+  main:
+    enabled: false
+    ports:
+      main:
+        enabled: false
+
+zammad:
+  zammad:
+    zammad_autowizardjson: ""
+  webconcurrency:
+    webcon_instances: ""
+    webcon_sessionjobs: 0
+    webcon_schedprocesses: false
+    webcon_delayedjobs: 0
+  rails:
+    rails_trustedproxies: ['127.0.0.1', '::1']
```

Comment:
Your lint is failing here btw

Reply:
this is how its done... use [] for empty list.
```yaml
rails_trustedproxies:
   - 127.0.0.1
```

Reply:
Got it, added the IPv4 and IPv6 addresses in that list in both locations. Thanks guys!